examples/digital-ocean-api.ts

```diff
@@ -3483,7 +3483,7 @@ export interface external {
      * @example Sammy Shark
      */
     user_name?: string;
-    user_billing_address?: Record<string, never> & external["resources/billing/models/billing_address.yml"];
```

Comment:
This is an intentional change - since `unknown & AnyType === AnyType` is a tautology, I've added code to tsIntersectionOf to remove these redundant bits of the intersection.

These extraneous unknowns are caused by the definitions in the schema:
```yaml
user_billing_address:
  allOf:
    - description: some description
    - $ref: 'billing_address.yml'
```

We (correctly) interpret this as two schemas, the first registers to be a schema with no type defined and therefore `unknown`. Why they did this escapes me.